rust/cubesqlplanner/cubesqlplanner/src/plan/join.rs

```diff
@@ -247,6 +278,7 @@ impl JoinCondition {
             JoinCondition::ToDateRollingWindowJoinCondition(cond) => {
                 cond.to_sql(templates, context)
             }
+            JoinCondition::RollingTotalJoinCondition(cond) => cond.to_sql(templates, context),
```

Comment:
If all conditions implement common trait with `to_sql()` - you could just call it here....